bom/application/pom.xml

```diff
@@ -6489,11 +6482,6 @@
                 <version>${mime4j.version}</version>
             </dependency>
             <!-- OpenTelemetry libs not in BOM -->
-            <dependency>
-                <groupId>io.opentelemetry.semconv</groupId>
-                <artifactId>opentelemetry-semconv</artifactId>
-                <version>${opentelemetry-semconv.version}</version>
-            </dependency>
             <dependency>
                 <groupId>io.opentelemetry.semconv</groupId>
                 <artifactId>opentelemetry-semconv-incubating</artifactId>
```

Comment:
From what I can see, this one is also defined in the BOM.

Reply:
I missed it.

Reply:
ummm... `opentelemetry-semconv-incubating` is not part of that bom

Reply:
Ah, actually, they added it in 2.9.0, you're right it's not there in 2.8.0. We can drop it when we upgrade.

Reply:
Cool! will do.